liquibase/changes/feat_240.sql

```diff
@@ -0,0 +1,26 @@
+-- 1. Add Table ValidationReport
+CREATE TABLE ValidationReport (
+    id VARCHAR(255) PRIMARY KEY,
+    dataset_id VARCHAR(255) REFERENCES GTFSDataset(id),
+    version VARCHAR(255)
+);
+
+-- 2. Modify Table GTFSDataset to add FK to ValidationReport
+ALTER TABLE GTFSDataset
+ADD COLUMN latest_validation_report VARCHAR(255) REFERENCES ValidationReport(id);
+
+-- 3. Create custom type for severity
+CREATE TYPE severity_type AS ENUM ('ERROR', 'WARNING', 'INFO');
+
+-- 4. Add Table Notice
+CREATE TABLE Notice (
+    dataset_id VARCHAR(255) REFERENCES GTFSDataset(id),
+    validation_report_id VARCHAR(255) REFERENCES ValidationReport(id),
+    notice_code VARCHAR(255),
+    severity severity_type,
+    PRIMARY KEY (dataset_id, validation_report_id, notice_code)
+);
```

Comment:
Please add the total notices field:
```suggestion
CREATE TABLE Notice (
    dataset_id VARCHAR(255) REFERENCES GTFSDataset(id),
    validation_report_id VARCHAR(255) REFERENCES ValidationReport(id),
    notice_code VARCHAR(255),
    severity severity_type,
    total_notices INT NOT NULL CHECK (total_notices > 0),
    PRIMARY KEY (dataset_id, validation_report_id, notice_code)
);
```

Reply:
should it be within the `Notice` entity?

Reply:
This field represents the total number of notices of that notice code in the report. That's why it should be in the Notice entity. To illustrate, find below a snipped of a  report:
```
{
  "summary": {
.....
 },
  "notices": [
    {
      "code": "block_trips_with_overlapping_stop_times",
      "severity": "ERROR",
      "totalNotices": 470,  <=== This is the field
      "sampleNotices": [.....]
}]
}
```